Makefile

```diff
@@ -192,8 +172,16 @@ models/arwiki.reverted.gradient_boosting.model: \
 		--pop-rate "false=0.9648134044173649" \
 		--center --scale > $@
 
+stats/arwiki.reverted.gradient_boosting.model_info.txt: \
+		models/arwiki.reverted.gradient_boosting.model
+	revscoring model_info $< > $@
+
+arwiki_reverted_model: \
+	models/arwiki.reverted.gradient_boosting.model \
+	stats/arwiki.reverted.gradient_boosting.model_info.txt
```

Comment:
This feels weird.  Maybe we should have a rule like `arwiki_model_infos` to match `arwiki_models` and `arwiki_tuning_reports`.

Reply:
I feel weird about this too, but what I wanted to accomplish was to build the model_info file any time a model was rebuilt.  If you go "make models" or "make arwiki_models", I want the model_info file to be rebuilt after the models.  This wrapping rule was the only way I could think to approximate that behavior.

Reply:
Aha!  How about something like this:
```
models/arwiki.reverted.gradient_boosting.model:
    revscoring cv_train <blah blah> > $@;
    revscoring model_info $@ > model_info/... 
```

That way the info is updated as part of the model building process.

Reply:
Hmm.  Not sure if that is abusive.  I think multiple outputs are common.  See latex.  Not sure if that's a *good* example though :\